plugins/package-managers/conan/src/main/kotlin/Conan.kt

```diff
@@ -73,8 +73,9 @@ internal class ConanCommand(private val useConan2: Boolean = false) : CommandLin
 
     override fun transformVersion(output: String) =
         // Conan could report version strings like:
-        // Conan version 1.18.0
-        output.removePrefix("Conan version ")
+        // \033[0;32mConan version 1.18.0\033[0;m
+        // Conan version 2.18.0
+        output.substringAfter("Conan version ")
```

Comment:
Hmm, but does this keep the "\033[0;m" after "1.18.0"?

Anyway, this should never happen as we do not output to a TTY here. I found https://github.com/conan-io/conan/issues/7658 to be related, and I'd prefer to set the `CONAN_COLOR_DISPLAY=0` environment variable. Or more generally set `NO_COLOR=1` when getting the version as I've just proposed at https://github.com/oss-review-toolkit/ort/pull/10501.

Reply:
>  ... I'd prefer to set the CONAN_COLOR_DISPLAY=0 environment variable. 

Just a thought, as this was quite a weird issue to pin down: If the environment variable is not set, it is quite tough for the user to figure out that it is indeed the color that is the problem and that this color feature should be disabled. If one does not use the container this might be not as obvious.

Reply:
>>    Yes but that gets filtered in later stages.
>
>I'm not sure where / how...

Maybe it is not the right way to put it. The later steps of version string processing seem unaffected by the color coding remains.

Reply:
> If one does not use the container this might be not as obvious.

I wasn't planning to set the environment variable in the Docker container, but [in ORT's `getVersion()` implementation](https://github.com/oss-review-toolkit/ort/pull/10501/files#diff-eab4d40ecb62a05d5313a25acd7e57d9bc9f45b959f5f271ffdddec16c192743R91).

> The later steps of version string processing seem unaffected by the color coding remains.

While it's correct that the (suffix of) the version string is not relied on it code, it would still show odd in `ort requirements`.